common/IoT-Introduction.md

```diff
@@ -1,3 +1,101 @@
 ********
-# TBD
-********
\ No newline at end of file
+# 1. Introduction
+The Internet of Things or IoT is a giant network of “Things”. “Things” can refer to a wide variety of devices. physical devices, vehicles, home appliances. IoT enables these objects to connect and exchange data. IoT is influencing our lifestyle. From droids you control with the smartphone to fire alarm sensor in your room, or your smartwatch which is tracking your body status. It is almost everything that you came across in daily life. 
+
+![](images/CM-IoT-Introduction/2019-12-20-16-54-48.png)
+
+The extensive set of applications for IoT devices is often divided into smart home, consumer, commercial and industrial. The smart home is an excellent start point to understand IoT. 
+
+# 2. Smart Home
+IoT devices are a part of the larger concept of home automation, which can include lighting, heating and air conditioning, media and security systems. Long-term benefits could include energy savings by automatically ensuring lights and electronics are turned off.  
+
+A smart home or automated home could be based on a platform or hub that controls smart devices and appliances. For instance, using Amazon Echo Plus to control the light, air conditioners, or sensing the room temperature. And the same way you can do with your smartphone app.  
+
+![](images/CM-IoT-Introduction/2019-12-17-17-26-03.png)
+
+In the smart home applications, the IoT environment consists of three parts:
+* __End Devices__  
```

Comment:
Add some simple inrodcution about end device and gateway and the differences, also inlcuding why we need gateway.